lib/dishes.rb

```diff
@@ -0,0 +1,25 @@
+class Dishes
+  attr_reader :name, :price, :available, :list_available_dishes
+
+  def initialize(name, price, available = true)
+    $name = name
+    $price = price
+    @available = available
+    @available_dishes = {}
+  end
+
+  def available?(availability)
+    if availability == "yes"
+      @available = true
+    elsif availability == "no"
+      @available = false
+    else
+      @available
+    end
+  end
+
+  def list_available_dishes
```

Comment:
Does a single dish need to know about a list_of_available dishes? - that feels like a job for a higher up class. Was this function moved and accidentally left over? I can see the Takeaway class has an attribute for this.